### PR TITLE
Update: new blog post dat grant

### PR DESCRIPTION
**Pull request by dpaez**
This PR adds a new blog post. It also touches the metadata, updating some values, that's why there are lot of files modified.

### CODE REVIEW

File: docs/feed/feed.xml [hunk 1]
```diff
@@ -1,25 +1,61 @@
...
 		<name>Dat Consortium</name>
 		<email>dat-consortium@protonmail.com</email>
 	</author>
 	
+	<entry>
+		<title>Dat Ecosystem Relaunch
+</title>
+		<link href="https://blog.datproject.org/dat-ecosystem-relaunch/"/>
+		<updated>2021-12-21T10:00:00Z</updated>
+		<id>https://blog.datproject.org/dat-ecosystem-relaunch/</id>
```
Comment (serapath) [inline docs/feed/feed.xml:20]: i think the URL here is outdated.
Maybe thats a problem that occures in many places.
Is that easy to fix?

File: docs/sitemap.xml [hunk 2]
```diff
@@ -326,73 +326,88 @@
     <lastmod>2020-08-22</lastmod>
   </url>
   
+  <url>
+    <loc>https://blog.datproject.org/dat-ecosystem-relaunch/</loc>
```
Comment (serapath) [inline docs/sitemap.xml:330]: same thing here

File: posts/2021-12-21-dat-grant-ecosystem-relaunch.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+___
+
+## A Little Context
+
+Around mid 2019, the Dat protocol was renamed to the Hypercore Protocol, i.e: `hyper://`. This was not only a technical change, but also affected the organizational structure (such as which Github organizations should hold which repositories, and with what responsibilities). Ultimately these changes helped communicate  boundaries and responsibilities: [hypercore-protocol.org](https://hypercore-protocol.org/) with (@mafintosh et al) will be in charge of all the core technical decisions.
+
+Great! But, what about the community? A group of projects within the community moved to create the `dat ecosystem` as a place to gather, discuss, and share projects built on Hypercore. This movement also spawned the `dat consortium`.
+
+If you're looking for more information about this change, it's been written about in [more detail](https://blog.datproject.org/dat-protocol-renamed-hypercore-protocol/) [previously](https://www.geutstudio.com/blog/hello-hyper/).
```
Comment (serapath) [inline posts/2021-12-21-dat-grant-ecosystem-relaunch.md:31]: same with the link here - its outdated

**Review by serapath — CHANGES_REQUESTED** [review]

File: docs/feed/feed.xml (hunk 1 above)
Comment (dpaez) [reply docs/feed/feed.xml:20]: Where do we link to? `https://blog.dat-ecosystem.org/` right?

**Comment by dpaez** [comment]
Links Updated

**Review by serapath — APPROVED** [review]

**Comment by serapath** [comment]
nice. then lets merge it i guess, no? :-)

**Pull request merged**